code/queue.rb

```diff
@@ -398,15 +398,15 @@ def run_queue_script!(msg)
           $log.debug("set ENV, now executing #{script_path}")
 
           # bash -lc will execute the command but first re-initializing like a new login (reading .bashrc, etc.)
-          exec_prefix = @config.exec_prefix || "bash -lc "
+          exec_prefix = @config.exec_prefix || "bash -lc"
```

Comment:
I didn't trace `@config.exec_prefix` but it should never be touching `script_path`?